bazel/kotlin_test.bzl

```diff
@@ -30,17 +30,18 @@ def _internal_kt_test(name, srcs, deps = [], data = [], jvm_flags = []):
 
 # A basic macro to make it easier to declare and run kotlin tests which depend on a JNI lib
 # This will create the native .so binary (for linux) and a .jnilib (for OS X) look up
-def envoy_mobile_jni_kt_test(name, srcs, native_deps = [], deps = []):
+def envoy_mobile_jni_kt_test(name, srcs, native_deps = [], deps = [], library_path = "library/common/jni", repository = ""):
```

Comment:
Pulling the library path variable up as well since parent repositories have a different relative library path than envoy-mobile